src/Modules/StatCan.OrchardCore.EmailTemplates/Controllers/EmailTemplateController.cs

```diff
@@ -213,12 +208,7 @@ public async Task<IActionResult> Edit(string sourceName, EmailTemplateViewModel
 
             var templatesDocument = await _templatesManager.LoadEmailTemplatesDocumentAsync();
 
-            if (ModelState.IsValid && !model.Name.Equals(sourceName, StringComparison.OrdinalIgnoreCase) && templatesDocument.Templates.ContainsKey(model.Name))
-            {
-                ModelState.AddModelError(nameof(EmailTemplateViewModel.Name), S["A email template with the same name already exists."]);
```

Comment:
Should we leave this validation ?

Reply:
With the Id it doesn't matter anymore unless we don't want them to have the same name for other reasons ?

Reply:
I think it might be to protect the user from itself. If you have two templates with the same name, it might be confusing ?

Reply:
fair enough